pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -575,21 +569,10 @@ async def _rebase_images(self):
 
     async def _mass_rebuild(self, doozer_cmd: list):
         lock = Lock.MASS_REBUILD
-        lock_manager = locks.LockManager.from_lock(lock)
-        lock_name = lock.value  # version agnostic, only 1 mass rebuild allowed
-
-        try:
-            # Try to acquire mass-rebuild lock for build version
-            async with await lock_manager.lock(lock_name):
-                self.runtime.logger.info('Running command: %s', doozer_cmd)
-                await exectools.cmd_assert_async(doozer_cmd)
-
-        except LockError as e:
-            self.runtime.logger.error('Failed acquiring lock %s: %s', lock_name, e)
-            raise
-
-        finally:
-            await lock_manager.destroy()
+        self.runtime.logger.info('Running command: %s', doozer_cmd)
+        await locks.run_with_lock(coro=exectools.cmd_assert_async(doozer_cmd),
+                                  lock=lock,
+                                  lock_name=lock.value.format(version=self.version.stream))
```

Comment:
Is the mass rebuild lock taking in version var?

Reply:
Good catch! Fortunately, the template string has no version param in it, so this `format` statement would not have any effect. Nonetheless, updated with https://github.com/openshift-eng/aos-cd-jobs/pull/3886/commits/6d6feb7bbae42509fc9c0f5a95304aa6659d3242